lib/geordi/commands/cucumber.rb

```diff
@@ -55,6 +55,7 @@ def cucumber(*args)
 
     invoke_geordi 'bundle_install'
     invoke_geordi 'yarn_install'
+    invoke_geordi 'chromedriver_update', show_warning: false
```

Comment:
Ok, we won't add a test because it is just too hard to fake the Chrome/chromedriver version determination.